canvas_modules/common-canvas/src/common-properties/controls/datepicker-range/datepicker-range.jsx

```diff
@@ -152,7 +148,7 @@ class DatepickerRangeControl extends React.Component {
 						onChange={this.handleInputStartChange.bind(this)}
 						value={this.state.valueStart}
 						onBlur={this.onStartBlur.bind(this)}
-						helperText={!this.props.tableControl && startHelperText}
```

Comment:
I don't think this is correct since there might be a different helper text for both.